api/workers/cleaner.go

```diff
@@ -22,15 +21,15 @@ import (
 func StartCleaner(_ context.Context, store store.Store) error {
 	envs, err := getEnvs()
 	if err != nil {
-		return fmt.Errorf("failed to get the envs: %w", err)
+		return err
 	}
 
 	if envs.SessionRecordCleanupRetention == 0 {
 		return nil
 	}
 
 	if envs.SessionRecordCleanupRetention < 0 {
-		return fmt.Errorf("invalid time interval: %w", fmt.Errorf("%d is not a valid time interval", envs.SessionRecordCleanupRetention))
+		return err
```

Comment:
In this case, the `err` does not exist, as nil, or the value does not correspond to the current problem.

I think we can maintain the `fmt.Errorf`.